server/src/Controller/Channel/ChannelController.js

```diff
@@ -0,0 +1,99 @@
+import {
+  create,
+  deleteDoc,
+  findOne,
+  findMany,
+  update,
+} from '../../Repositories/genericRepository';
+import successHandler from '../../libs/routes/successHandler';
+import { channelModel } from '../../Model';
+
+
+const getChannel = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const result = await findOne(channelModel, id);
+    if (!result) {
+      const err = new Error('Channel does not exists');
+      next(err);
+    }
+    res
+      .status(200)
+      .send(successHandler('Channel is here', result));
+  } catch (error) {
+    next(error);
+  }
+};
+
+const getAllChannels = async (req, res, next) => {
+  try {
+    const result = await findMany(channelModel);
+    res
+      .status(200)
+      .send(successHandler('Channels are here', result));
+  } catch (error) {
+    next(error);
+  }
+};
+
+const createChannel = async (req, res, next) => {
+  try {
+    const {
+      name,
+      workspace,
+      user,
+      isPrivate,
+      members,
+    } = req.body;
+    // Change this logic later accordingly
+    const membersArray = members.split(', ');
+    const data = {
+      name,
+      workspace,
+      user,
+      isPrivate,
+      members: membersArray,
+    };
+    const result = await create(channelModel, data);
+
+    res
+      .status(200)
```

Comment:
For Create APIs response code should be ideally `201`

Reply:
Okay, I'll change it.